test/dhtest.c

```diff
@@ -166,6 +166,9 @@ int main(int argc, char *argv[])
 
     alen = DH_size(a);
     abuf = OPENSSL_malloc(alen);
+    if (!abuf)
```

Comment:
Oh, and something else I missed first time through. Can you change that to `abuf == NULL` and the same for other places where pointers are used like a boolean? We've been trying to standardise on a consistent style for this.